src/raftstore/coprocessor/split_check/size.rs

```diff
@@ -16,38 +16,53 @@ use rocksdb::DB;
 use util::transport::{RetryableSendCh, Sender};
 
 use super::super::metrics::*;
-use super::super::{Coprocessor, ObserverContext, SplitCheckObserver, SplitChecker};
+use super::super::{Coprocessor, ObserverContext, RowEntry, SplitCheckObserver, SplitChecker};
 use super::Host;
 
 pub struct Checker {
     max_size: u64,
     split_size: u64,
+    max_rows: u64,
```

Comment:
You should create another `Checker` for rows.

Reply:
If add an independent Checker for rows, then it will get approximate stat and send it to raftstore again.
Or maybe we should separate `get_approximate_size` and `get_approximate_rows`? What do you think?